src/config/generalconf.cpp

```diff
@@ -777,11 +778,37 @@ void GeneralConf::initShowSelectionGeometry()
     vboxLayout->addStretch();
 }
 
+void GeneralConf::initJpegQuality()
+{
+    auto* tobox = new QHBoxLayout();
+
+    int timeout =
```

Comment:
Is the "timeout" variable name intentional? If not, I suggest changing it to something more fitting to improve code readability

Reply:
The fix has been pushed.